### PR TITLE
Add enketo encryption namespaces

### LINKED ISSUE #2121: Allow submission from enketo encrypted form
## Suggested Feature / Enhancement

Allow submission from enketo for an encrypted form

## Benefits of implementing the feature/enhancement
Support enketo  submission from an encrypted form.

## Suggested implementation plan(Steps to be taken to implement feature)

Add enketo encryption namespace to this config https://github.com/onaio/onadata/blob/master/onadata/libs/utils/logger_tools.py#L64

### PR DESCRIPTION
**Pull request by denniswambua**
### Before submitting this PR for review, please make sure you have:

- [ ] Included tests
- [ ] Updated documentation

Closes #2121

### CODE REVIEW

File: onadata/libs/utils/logger_tools.py [hunk 1]
```diff
@@ -62,7 +62,9 @@
 DEFAULT_CONTENT_LENGTH = settings.DEFAULT_CONTENT_LENGTH
 REQUIRED_ENCRYPTED_FILE_ELEMENTS = [
     "{http://www.opendatakit.org/xforms/encrypted}base64EncryptedKey",
-    "{http://www.opendatakit.org/xforms/encrypted}encryptedXmlFile"]
+    "{http://www.opendatakit.org/xforms/encrypted}encryptedXmlFile",
+    "{http://opendatakit.org/submissions}base64EncryptedKey",
+    "{http://opendatakit.org/submissions}encryptedXmlFile"]
```
Comment (DavisRayM) [inline onadata/libs/utils/logger_tools.py:67]: Are the new tags unique to only Enketo ? 

_There is a [check](https://github.com/onaio/onadata/blob/fd58c7e26b5c136b8b0915c3714f1beaebdb7bcb/onadata/libs/utils/logger_tools.py#L320-L325) to confirm that only `2` of the tags are present hopefully there isn't a scenario where more than two of these tags can be present in an XML Submission_

**Review by DavisRayM — APPROVED** [review]
LGTM just have a comment

File: onadata/libs/utils/logger_tools.py (hunk 1 above)
Comment (denniswambua) [reply onadata/libs/utils/logger_tools.py:67]: The two are the only elements we need for this encryption feature

Comment (DavisRayM) [reply onadata/libs/utils/logger_tools.py:67]: ODK Collect shares the same elements ?

Comment (denniswambua) [reply onadata/libs/utils/logger_tools.py:67]: They use the same elements but different namespaces. For odk it is `{http://www.opendatakit.org/xforms/encrypted}` while for enketo is `{http://opendatakit.org/submissions}`

Comment (DavisRayM) [reply onadata/libs/utils/logger_tools.py:67]: Ohh cool, makes sense

**Pull request merged**